readme.md

```diff
@@ -1061,10 +1061,21 @@ If you want to create re-usable React components that you can embed in your Next
 <details>
   <summary>How do I use CSS-in-JS solutions?</summary>
 
-Next.js bundles [styled-jsx](https://github.com/zeit/styled-jsx) supporting scoped css. However you can use a CSS-in-JS solution in your Next app by just including your favorite library [as mentioned before](#css-in-js) in the document.
+Next.js bundles [styled-jsx](https://github.com/zeit/styled-jsx) supporting scoped css. However you can use any CSS-in-JS solution in your Next app by just including your favorite library [as mentioned before](#css-in-js) in the document.
 </details>
 
 <details>
+  <summary>How do I CSS preprocessors like SASS / SCSS / LESS?</summary>
+
+Next.js bundles [styled-jsx](https://github.com/zeit/styled-jsx) supporting scoped css. However you can use any CSS preprocessor solution in your Next app by following one of these examples:
+
+- [with-external-scoped-css](./examples/with-external-scoped-css)
+- [with-global-stylesheet](./examples/with-global-stylesheet)
```

Comment:
Also this example https://github.com/zeit/next.js/tree/master/examples/with-scoped-stylesheets-and-postcss 👍